recipes/sqlite3/all/conanfile.py

```diff
@@ -86,6 +89,10 @@ def configure(self):
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
 
+        if (not self.options.enable_default_vfs) and self.options.build_executable:
```

Comment:
```suggestion
    def validate(self):
        if not self.options.enable_default_vfs and self.options.build_executable:
```

Reply:
Looks like `validate()` method is experimental feature: [see here](https://docs.conan.io/en/latest/reference/conanfile/methods.html#validate). That is why I don't want to use it.

Reply:
It's important to note, CCI is the test platform for new Conan features so we want to use them.

Rest assured it's safe to use... there's a lot of it already deployed.